docs/admission-control-secure.md

```diff
@@ -70,172 +77,77 @@ Make sure that the minikube ingress addon is enabled:
 minikube addons enable ingress
 ```
 
-### 2. Create a new Namespace to deploy OPA into
-
-When OPA is deployed on top of Kubernetes, policies are automatically loaded out of ConfigMaps in the `opa` namespace.
-
-```bash
-kubectl create namespace opa
-```
-
-Configure `kubectl` to use this namespace:
-
-```bash
-kubectl config set-context opa-tutorial --user minikube --cluster minikube --namespace opa
-kubectl config use-context opa-tutorial
-```
-
-### 3. Deploy OPA on top of Kubernetes
+Follow the steps in the [Kubernetes Admission Control](http://www.openpolicyagent.org/docs/kubernetes-admission-control.html) tutorial to create the `opa` namespace and configure TLS. Now use the **admission-controlller.yaml** file from the tutorial to deploy OPA as an admission controller with the following changes:
 
-Communication between Kubernetes and OPA must be secured using TLS. To configure TLS, use `openssl` to create a certificate authority (CA) and certificate/key pair for OPA:
+1. Use the below `opa` and `kube-mgmt` container spec which enables OPA's security features and configures `kube-mgmt` to include a `Bearer` token in calls to OPA. We also volume mount OPA's startup authorization policy `inject.rego` inside the OPA container in the `/policies` directory.
 
-```bash
-openssl genrsa -out ca.key 2048
-openssl req -x509 -new -nodes -key ca.key -days 100000 -out ca.crt -subj "/CN=admission_ca"
-```
-
-Generate the TLS key and certificate for OPA:
+```yaml
+spec:
+  containers:
+    - name: opa
+      image: openpolicyagent/opa:0.10.0
+      args:
+        - "run"
+        - "--server"
+        - "--tls-cert-file=/certs/tls.crt"
+        - "--tls-private-key-file=/certs/tls.key"
+        - "--addr=0.0.0.0:443"
+        - "--addr=http://127.0.0.1:8181"
+        - "--authentication=token"
+        - "--authorization=basic"
+        - "/policies/inject.rego" # authorization policy used on startup
```

Comment:
Nit: let's call this something other inject.rego, e.g., authz.rego.

Reply:
Changed `inject.rego` to `authz.rego`.